python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -98,7 +101,7 @@ def _setNamePath(self, _namePattern='', _localDir=''):
     def _on_attribute__set__(self, obj_type, attrib_name):
         # This is defining the object as uncopyable from outputfiles... do we want this mechanism still?
         r = copy.deepcopy(self)
-        if getName(obj_type) == 'Job' and attrib_name == 'outputfiles':
+        if _getName(obj_type) == 'Job' and attrib_name == 'outputfiles':
```

Comment:
Yes this probably could be. I'm always hesitant about touching the filters as I'm not certain where the proxy is/was stripped in the process. Given the new tests are working pretty well I'll see that this doesn't break anything and push the change 👍